orangecontrib/text/widgets/owconcordance.py

```diff
@@ -256,7 +280,7 @@ def set_word_from_input(self, topic):
     def set_word(self):
         self.model.set_word(self.word)
         self.update_widget()
-        self.commit()
+        self.set_selection()
```

Comment:
Why this? Currently, if I am inputing different words on input the selection persists which is kind on unintuitive?